components/token-container.tsx

```diff
@@ -18,11 +21,44 @@ type TokenContainerProps = FlexProps & {
   setAmount: Dispatch<SetStateAction<string>>;
   setFocused: Dispatch<SetStateAction<string>>;
   maxBalance?: string;
+  calculatePriceAndToken?: () => void;
 };
 
 export const TokenContainer = (props: TokenContainerProps) => {
   const ethCard = props.symbol === "ETH";
 
+  const maxValueWithGas = async () => {
+    if (!props.maxBalance || parseFloat(props.maxBalance) === 0) return "0";
+
+    const defaultGasEstimate = parseEther("0.0045");
```

Comment:
would be great creating a const for all of that and and import it from a central place... (for the numbers)

Reply:
Done